charts/flagsmith/templates/secrets-api.yaml

```diff
@@ -11,6 +11,41 @@ metadata:
     {{- include "flagsmith.labels" . | nindent 4 }}
     app.kubernetes.io/component: api
 type: Opaque
+stringData:
+  DATABASE_URL: {{ include "flagsmith.api.databaseUrl" . | trim | quote }}
+---
+{{/*
+ If no secrets were provided in values, check if a previously deployed secret exists
+ If none exists, create one, which will be reused in future deployments
+ */}}
+{{- $secretKey := default (randAlphaNum 50) .Values.api.secretKey -}}
```

Comment:
I don't think `randAlphaNum` is cryptographically secure?

Reply:
It is actually! https://helm.sh/docs/chart_template_guide/function_list/#randalphanum-randalpha-randnumeric-and-randascii

> randAlphaNum, randAlpha, randNumeric, and randAscii
>
> These four functions generate cryptographically secure (uses crypto/rand) random strings, but with different base character sets